lib/Base/Utils/StateIO.vala

```diff
@@ -0,0 +1,61 @@
+using GLib;
+
+namespace Backend.Utils.StateIO {
```

Comment:
```suggestion
/* StateIO.vala
 *
 * Copyright 2023 IBBoard
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 * SPDX-License-Identifier: GPL-3.0-or-later
 */

using GLib;

/**
 * Provides methods to load and store the state variants.
 */
namespace Backend.Utils.StateIO {
```

Copyright header was missing.